nbtc/relayer.go

```diff
@@ -20,12 +21,21 @@ type Relayer struct {
 	btcClient        bitcoin.Client
 	shutdownChan     chan struct{}
 	processTxsTicker *time.Ticker
+	confirmTxsTicker *time.Ticker
 }
 
 // NewRelayer creates a new Relayer instance with the given configuration.
-func NewRelayer(btcClientConfig rpcclient.ConnConfig, processTxsInterval time.Duration, db *dal.DB) (*Relayer, error) {
+func NewRelayer(
+	btcClientConfig rpcclient.ConnConfig,
```

Comment:
Group those parameters into 2 groups: config and db. Do you think this is more readable and maintainable?

Reply:
Feel free to do this in other PR.

Reply:
I grouped it in two different configs, leaving the db seperate